README.md

```diff
@@ -1,2 +1,39 @@
-# fastly-template-rust-nel
-Package template for a Rust based Network Error Logging Compute@Edge service
+# Compute@Edge starter kit for Network Error Logging
+
+A Rust based Compute@Edge stater kit for a [Network Error Logging][spec] reporting endpoint.
+
+> Network Error Logging is a mechanism that can be configured via the NEL HTTP response header. This experimental header allows web sites and applications to opt-in to receive reports about failed (and, if desired, successful) network fetches from supporting browsers.
+(source: [MDN][MDN])
+
+**For more details about this and other starter kits for Compute@Edge, see the [Fastly developer hub](https://developer.fastly.com/solutions/starters)**
+
+[spec]: https://w3c.github.io/network-error-logging/
+[MDN]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging
+
+## Usage
+- Install the [Fastly CLI][latest] to your `$PATH`
+- Configure the CLI with an API token via `fastly configure`
+- Run `fastly compute init --from https://github.com/fastly/fastly-template-rust-nel.git`
+
+[cli]: https://github.com/fastly/cli
+[latest]: https://github.com/fastly/cli/releases/latest
+
+## Features
+
+* Exposes a `POST /reports` endpoint to receive NEL reports
+* Deserializes individual reports from JSON to Rust data structures
+* Adds additional information at the edge such as geo IP data
+* Sends reports to a logging endpoint as individual JSON lines 
+* Responds with a synthetic 204 response from the edge
+
+## Requirements:
+The following resources need to exist on your active service version for the 
+service to work end-to-end.
+
+- A [logging endpoint][logging] called `reports`.
+
+[logging]: https://docs.fastly.com/en/guides/about-fastlys-realtime-log-streaming-features
```

Comment:
The way we're currently ingesting these READMEs is to remove everything prior to the first subheading (to allow for more repo-flavoured content and a link to the developer hub), so that section should not contain anything that you want to display publicly.

Installation instructions are generic to all starters so we generate them in the developer hub. The link to the devhub should cover that off.

Since parsing of these files removes certain sections, it's a bit dangerous to have decoupled link references.  I'm proposing that we do all links inline.

I'm also feeling like the titles of these are a bit repetitive, but we can maybe consider that later.

```suggestion
# Compute@Edge starter kit for Network Error Logging

A Rust based Compute@Edge starter kit for a [Network Error Logging](https://w3c.github.io/network-error-logging/) reporting endpoint.

**For more details about this and other starter kits for Compute@Edge, see the [Fastly developer hub](https://developer.fastly.com/solutions/starters)**

## What is network error logging?

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging):

> Network Error Logging is a mechanism that can be configured via the `NEL` HTTP response header. This experimental header allows web sites and applications to opt-in to receive reports about failed (and, if desired, successful) network fetches from supporting browsers.

Our starter kit will bootstrap a Fastly service that can receive network error reports, parse them, enrich them with information available at the edge, and dispatch them to your logging provider.

## Features

* Exposes a `POST /reports` endpoint to receive NEL reports
* Deserializes individual reports from JSON to Rust data structures
* Adds additional information at the edge, such as geo IP data
* Sends reports to a logging endpoint as individual JSON lines 
* Responds with a synthetic 204 response to the client

## Requirements

The following resources need to exist on your active Fastly service version for this starter kit to work:

- A [logging endpoint](https://docs.fastly.com/en/guides/about-fastlys-realtime-log-streaming-features) called `reports`.
```